ui/seed_backup_page.go

```diff
@@ -0,0 +1,538 @@
+		info:  c.info,
+
+		action:         c.theme.Button("View seed phrase"),
+		backButton:     c.theme.PlainIconButton(c.icons.navigationArrowBack),
+		title:          c.theme.H5("Keep in mind"),
+		steps:          c.theme.Body1("Step 1/2"),
+		instruction:    c.theme.H6("Write down all 33 words in the correct order"),
+		successMessage: c.theme.H4("Your seed phrase backup is verified"),
+		successInfo:    c.theme.Body2("Be sure to store your seed phrase backup in a secure location."),
+		checkIcon:      c.icons.actionCheckCircle,
+
+		backButtonWidget: new(widget.Button),
+		actionWidget:     new(widget.Button),
+
+		active:        infoView,
+		selectedSeeds: make([]string, 0, 33),
+	}
+
+	b.checkIcon.Color = c.theme.Color.Success
+	b.steps.Color = c.theme.Color.Hint
+	b.successMessage.Alignment = text.Middle
+	b.successInfo.Alignment = text.Middle
+	b.successInfo.Color = b.theme.Color.Hint
+
+	b.backButton.Color = c.theme.Color.Hint
+	b.backButton.Size = unit.Dp(32)
+
+	b.action.Background = c.theme.Color.Hint
+
+	b.checkBoxes = []decredmaterial.CheckBox{
+		c.theme.CheckBox("The 33-word seed phrase is EXTREMELY IMPORTANT."),
+		c.theme.CheckBox("Seed phrase is the only way to restore your wallet."),
+		c.theme.CheckBox("It is recommended to store your seed phrase in a physical format (e.g. write down on a paper)."),
+		c.theme.CheckBox("It is highly discouraged to store your seed phrase in any digital format (e.g. screenshot)."),
+		c.theme.CheckBox("Anyone with your seed phrase can steal your funds. DO NOT show it to anyone."),
+	}
+
+	b.instruction.Alignment = text.Middle
+	b.allSuggestions = dcrlibwallet.PGPWordList()
+
+	for _, cb := range b.checkBoxes {
+		cb.IconColor = c.theme.Color.Success
+		cb.Color = c.theme.Color.Success
+	}
+
+	for i := 0; i < len(b.checkBoxes); i++ {
+		b.checkBoxWidgets = append(b.checkBoxWidgets, new(widget.CheckBox))
+	}
+
+	for i := 0; i < 33; i++ {
+		var bg []buttonGroup
+
+		for j := 0; j < 3; j++ {
+			bg = append(bg, buttonGroup{
+				skin:   c.theme.Button(""),
+				button: new(widget.Button),
+			})
+		}
+		b.suggestions = append(b.suggestions, seedGroup{selected: -1, buttons: bg})
+		b.selectedSeeds = append(b.selectedSeeds, "-")
+	}
+
+	b.infoList = &layout.List{Axis: layout.Vertical}
+	b.viewList = &layout.List{Axis: layout.Vertical}
+	b.seedPhraseListLeft = &layout.List{Axis: layout.Vertical}
+	b.seedPhraseListRight = &layout.List{Axis: layout.Vertical}
+	b.verifyList = &layout.List{Axis: layout.Vertical}
+	b.suggestionList = &layout.List{Axis: layout.Horizontal}
+
+	return func() {
+		b.layout()
+		b.handle(c)
+	}
+}
+
+func (pg *backupPage) layout() {
+	pg.theme.Surface(pg.gtx, func() {
+		toMax(pg.gtx)
+		layout.Flex{Axis: layout.Vertical, Alignment: layout.Start}.Layout(pg.gtx,
+			layout.Rigid(func() {
+				pg.action.Background = pg.theme.Color.Hint
+				switch pg.active {
+				case infoView:
+					if pg.verifyCheckBoxes() {
+						pg.action.Background = pg.theme.Color.Primary
```

Comment:
`pg.action.Background = pg.theme.Color.Primary` is repeated about 3 times sames as `pg.action.Color = pg.theme.Color.InvText`

These 2 can be added to `updateActionBackground()` and `updateActionColor()` respectively and called where needed.